simulation/revolve2/simulation/simulator/_batch_parameters.py

```diff
@@ -29,3 +29,9 @@ class BatchParameters:
 
     control_frequency: float
     """Similar to `sampling_frequency` but for how often the control function is called."""
+
+    cast_shadows: bool
```

Comment:
I think these parameters would be better fit as arguments to the LocalSimulator __init__, next to `headless`